### ISSUE TITLE
Theme switcher light/dark toggle

### ISSUE THREAD

**Issue by tijlleenders** [tijlleenders/ZinZen#1167]
**Describe the bug**
When selecting 'Change theme' the switch mode button in bottom switcher menu is always selected.
It is also not clear you can press it. Not a single test user pressed it, but they did use the left and right arrows.

**To Reproduce**
Ask anyone to use the 'Change theme' option without telling them anything and observe.

**Expected behavior**
I should understand without explanation the middel button can be pressed.  
Show a toggle with a sun and a moon would make this better I believe. We can use the one we had previously?

**Screenshots**
<img width="224" alt="image" src="https://user-images.githubusercontent.com/44887755/235810547-b68c2f40-abc4-47ea-8ca4-607c59379c97.png">

**Comment by Tushar-4781** [comment]
You mean the text "Switch Mode" is not enough ?

**Comment by tijlleenders** [comment]
No, that is not enough. Nobody I talked to tried pressing the button. They just used left and right. Maybe they think they're already in 'switch mode': like that is the title of the menu - after preasing 'Change theme'.

**Comment by vinaybadgujar102** [comment]
We can replace "Switch Mode" with "Toggle Dark Mode" / "Toggle Light Mode" text?

**Comment by tijlleenders** [comment]
@vinaybadgujar102 
That would indeed be a better text, thanks. This will not be sufficient to resolve this issue - which deals with "inattentional blindness".
